src/test/regress/expected/coordinator_shouldhaveshards.out

```diff
@@ -375,13 +375,47 @@ NOTICE:  executing the command locally: SELECT a.count, b.x, b.y, c.a, c.b, d.co
    100 | 3 | 2 | 3 | 2 |     1
 (1 row)
 
+-- issue 3802
+SET citus.shard_replication_factor TO 2;
+CREATE TABLE dist_table(a int);
+ERROR:  relation "dist_table" already exists
+SELECT create_distributed_table('dist_table', 'a');
+NOTICE:  Copying data from local table...
+NOTICE:  copying the data has completed
+DETAIL:  The local data in the table is longer visible, but is still on disk.
+HINT:  To remove the local data, run: SELECT truncate_local_data_after_distributing_table($$coordinator_shouldhaveshards.dist_table$$)
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+BEGIN;
+-- use transtion block so that we will call ExtractLocalAndRemoteTasks
```

Comment:
I would prefer to add such a comment instead of referring to a function name

```suggestion
-- force using local execution using transaction block to test if
-- we create remote & local tasks properly
```

Reply:
Here we test the case where the task query will be `perPlacementQueryStrings`, so maybe something better could be:

```
-- this will use perPlacementQueryStrings, make sure it works correctly with
-- copying task
```